polus-czi-extract-plugin/Dockerfile

```diff
@@ -0,0 +1,125 @@
+### 1. Get Linux
```

Comment:
I think a lot of this Dockerfile can be condensed. The Java installation seems needlessly complex, and I was able to simplify the build process by using the `openjdk:11-slim` base and installing python3 using: `apt-get install -y --no-install-recommends python3-pip python3-setuptools python3.7-dev`. I have not tested running the application, but the dependency installation worked without issue. I will also keep looking at an alpine build when I have the chance.

Reply:
The main reason I ended on this Dockerfile configuration is because of issues I was having trying to get the application to run properly. However, as you see in the commits, there was a bad package that might have been the source of the problem. I'll try to simplify the image and base it on Alpine to reduce the image size (currently ~900MB).